wiki/Building-From-Source.md

```diff
@@ -8,6 +8,21 @@ You need to install [rustup](https://www.rust-lang.org/tools/install).
 
 On Windows you need also [Chocolatey](https://chocolatey.org/install).
 
+# Users with Nvidia GPUs
+
+Before building the server, those with nvidia GPUs would have to build and install [`FFmpeg/nv-codec-headers`](https://github.com/FFmpeg/nv-codec-headers). The nv-codec-headers requires at least the following driver version for the respective OS to work properly, taken from nv-codec-header's README:
```

Comment:
Please specify this is for Linux only. Also people on AMD that omit `--no-nvidia` need to install this, so it's better to make it a requirement regardless.

Reply:
How's this?

> Before building the server, those on Linux with would have to build and install [`FFmpeg/nv-codec-headers`](https://github.com/FFmpeg/nv-codec-headers). The nv-codec-headers, for nvidia users at least, requires driver version `520.56.06` or later to work properly; Taken from nv-codec-header's README.

Reply:
Yeah that looks fine